site/security_users_groups_roles.md

```diff
@@ -0,0 +1,53 @@
+### Modeling WebLogic Users, Groups and Roles
```

Comment:
WebLogic Users, Groups and Roles -> WebLogic Users, Groups, (comma) and Roles (our style guide dictates the use of the Oxford (serial) comma: in a list of three or more items, separate the items with commas, including a comma before the conjunction. Please make this correction globally.

Reply:
This and the other comments have been addressed.